src/arche/tools/schema.py

```diff
@@ -134,13 +134,13 @@ def format_validation_message(
     error_msg: str, path: Deque, schema_path: Deque, validator: str
 ) -> str:
     str_path = "/".join(p for p in path if isinstance(p, str))
-    schema_path = "/".join(p for p in schema_path)
+    schema_path_str: str = "/".join(p for p in schema_path)
```

Comment:
Why the variable was renamed?

Reply:
at the function declaration `schema_path` has type `Deque`, but at this line, `join` returns a `str`